packages/connection-model/package.json

```diff
@@ -23,14 +23,17 @@
     "test-ci": "npm run test",
     "posttest-ci": "node ../../scripts/killall-mongo.js"
   },
+  "peerDependencies": {
+    "mongodb": "4.x"
+  },
   "dependencies": {
     "@mongodb-js/ssh-tunnel": "^1.2.0",
     "ampersand-model": "^8.0.0",
     "ampersand-rest-collection": "^6.0.0",
     "async": "^3.1.0",
     "debug": "^4.1.1",
     "lodash": "^4.17.15",
-    "mongodb": "^3.6.3",
+    "mongodb3": "npm:mongodb@^3.6.3",
```

Comment:
Could we 'copy-paste' the `parse-connection-string.js` file and drop this one? We need to rework that to switch it to the new module anyway.

Reply:
I think we could... but I don't think we'd exactly want that to be the long-term solution here either, it's probably best to do some actual URL parsing based on mongodb-connection-string-url and then actually handle all the transformations here in the connection model directly (assuming that we keep mongodb-connection-model)

Reply:
> but I don't think we'd exactly want that to be the long-term solution here either

Yeah i was thinking the same, we should definitely use the `mongodb-connection-string-url` module instead.